Onboarding.md

```diff
@@ -16,29 +18,39 @@ In this document we'll walk through the setup for the Mojaloop Central Ledger. I
 4. Docker
 5. PostgreSQL 9.4
 6. pgAdmin4
-7. Visual Studio Code
-8. Postman
-9. nvm
-10. npm
-11. Zenhub
-12. central_ledger
+7. Postman
+8. nvm
+9. npm
+10. Zenhub
+11. central_ledger
+12. JavaScript IDE
 ***
 
 ### Setup
 Make sure you have access to [Mojaloop on Github](https://github.com/mojaloop/central-ledger) and clone the project.
 
-To install Homebrew run this in a terminal window:
+#### Installing Slack (Optional):
```

Comment:
I still dont believe this should be here. It really has nothing to do with the running of Central Ledger.

Please remove it.